mouse.go

```diff
@@ -0,0 +1,46 @@
+package terminal
+
+import (
+	"fyne.io/fyne/v2"
+	"fyne.io/fyne/v2/driver/desktop"
+)
+
+func (t *Terminal) handleMouseDownV200(btn int, mods desktop.Modifier, pos fyne.Position) {
+	_, _ = t.Write(t.encodeMouse(btn, mods, pos))
```

Comment:
Hmm, I am curious as to why staticcheck doesn't complain about this. I think it's fine to just do `t.Write()` without the blank identifier assignments. This applies to the below items as well.

Reply:
If I leave out the assignment then the IDE warns about ignored errors